sdk/servicebus/Azure.Messaging.ServiceBus/src/Core/BasicRetryPolicy.cs

```diff
@@ -30,7 +30,7 @@ internal class BasicRetryPolicy : ServiceBusRetryPolicy
         ///   behavior.
         /// </summary>
         ///
-        public ServiceBusRetryOptions Options { get; }
+        public override ServiceBusRetryOptions Options { get; }
```

Comment:
Exposing this in ServiceBusRetryPolicy to provide access to the default timeout.